features/checkout/seeing_proper_checkout_name.feature

```diff
@@ -0,0 +1,33 @@
+@checkout
```

Comment:
File name could be adjusted from `seeing_proper_checkout_name.feature` to `seeing_purchaser_identifier_in_checkout_page.feature`